app/controllers/application_controller.rb

```diff
@@ -1,12 +1,11 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
-  before_action :set_current_user
 
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name posts_counter])
-    devise_parameter_sanitizer.permit(:account_update, keys: %i[name posts_counter])
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[email name password password_confirmation posts_counter photo])
```

Comment:
- [ ] Great job removing the checkbox to match the wireframe as recommended by the previous reviewer 👍🏽 

<img width="1258" alt="Screenshot 2023-05-17 at 00 11 11" src="https://github.com/anas599/blog/assets/58692590/1da6383c-ceeb-413b-8435-8fec4d746f90">